src/main/java/io/kestra/plugin/pulsar/Produce.java

```diff
@@ -36,12 +36,12 @@
 @Getter
 @NoArgsConstructor
 @io.swagger.v3.oas.annotations.media.Schema(
-    title = "Produce message in a Pulsar topic"
+    title = "Produce message to a Pulsar topic."
 )
 @Plugin(
     examples = {
         @Example(
-            title = "Read a csv, transform it to right format & produce it to Pulsar",
+            title = "Read a csv, transform it to right format & produce it to Pulsar.",
```

Comment:
```suggestion
            title = "Read a CSV file, transform it to the right format & load it to Pulsar.",
```

Reply:
changing it to the word "publish", rather than "load", as per Pulsar docs.